init.fish

```diff
@@ -3,7 +3,14 @@ function init -a path --on-event init_nvm
 
   if available fenv
     set -q NVM_DIR; or set -gx NVM_DIR ~/.nvm
-    fenv source $NVM_DIR/nvm.sh >/dev/null ^&1
+
+    if available brew; and brew --prefix nvm > /dev/null
+      set nvm_runnable (brew --prefix nvm)
```

Comment:
I would use a global variable here `set -g nvm_prefix (brew --prefix nvm)`, so that we don't need to check it again in `nvm` function.